WordPress/Classes/ViewRelated/Blog/JetpackSecuritySettingsViewController.swift

```diff
@@ -0,0 +1,334 @@
+import Foundation
+import CocoaLumberjack
+import WordPressShared
+
+
+/// The purpose of this class is to render and modify the Jetpack Settings associated to a site.
+///
+open class JetpackSecuritySettingsViewController: UITableViewController {
+
+    // MARK: - Private Properties
+
+    fileprivate var blog: Blog!
+    fileprivate var service: BlogJetpackSettingsService!
+    fileprivate lazy var handler: ImmuTableViewHandler = {
+        return ImmuTableViewHandler(takeOver: self)
+    }()
+
+    // MARK: - Computed Properties
+
+    fileprivate var settings: BlogSettings {
+        return blog.settings!
+    }
+
+    // MARK: - Static Properties
+
+    fileprivate static let footerHeight = CGFloat(34.0)
+    fileprivate static let learnMoreUrl = "https://jetpack.com/support/sso/"
+    fileprivate static let wordPressLoginSection = 2
```

Comment:
This hardcoded section makes me a bit uncomfortable but I don't really have any better ideas 😞

Reply:
ha ha yeah I don't like it either, sorry!